src/templates/next/components/CollectionSearch/CollectionSearch.tsx

```diff
@@ -0,0 +1,29 @@
+import { BiSearch } from "react-icons/bi"
+import { CollectionSearchProps } from "~/common"
+
+const CollectionSearch = ({
+  placeholder,
+  search,
+  setSearch,
+}: Omit<CollectionSearchProps, "type">) => {
+  return (
+    <label className="relative block">
+      <span className="sr-only">{placeholder}</span>
+      <span className="absolute inset-y-0 left-0 flex items-center pl-4">
```

Comment:
just curious why need to use relative and absolute for positioning the search icon

Reply:
The `relative` here is for the `absolute` positioning to know which parent element to position relative to. Otherwise it will position based on the document itself.

This part references https://tailwindcss.com/docs/hover-focus-and-other-states#placeholder-text

Reply:
oh i meant like doesn't padding solve that already, but nvm non-issue